src/components/result/CreateResult.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
+import { Result } from './Result'
```

Comment:
Could we change the naming here `Result` is very generic. Instead of using `Result` we can use something like `SurveyResult`.

Reply:
And if we decide to update name `Result` we should update it for everything not just for this data type we should update for functions etc.

Reply:
ı agree with yüksel